IdentitySdkCore/IdentitySdkCore/Classes/ReachFivePasswordLess.swift

```diff
@@ -41,10 +41,14 @@ public extension ReachFive {
     
     func verifyPasswordlessCode(verifyAuthCodeRequest: VerifyAuthCodeRequest) -> Future<AuthToken, ReachFiveError> {
         let pkce: Pkce? = storage.take(key: pkceKey)
+        guard let pkce else {
```

Comment:
These two lines can be merged (I think?) Is it this way because it's more readable ?

Reply:
En faisant ça ?
```suggestion
        guard let pkce: Pkce = storage.take(key: pkceKey) else {
```
Ça a l'air d'être possible, je ne sais pas si c'est plus clair.